src/index.js

```diff
@@ -0,0 +1,50 @@
+import './style.css';
+import Completed from './components/status.js' ;
+
+
+const ul = document.querySelectorAll("ul");
+const todos = JSON.parse(localStorage.getItem('todo')) || [
+  {
+    description: "Take a shower",
+    completed: false,
+    //index: 1,
+  },
+  {
+    description:"Read my new book",
+    completed: false,
+    //index: 2,
+  },
+  {
+    description: "Wash asdfasdfasdfasdf dishes",
+    completed: true,
+    //index: 3,
+  },
+  {
+    description: "Go at the supermarket",
+    completed: true,
+    //index: 4,
+  },
+];
+
+const displayTodos = () => {
+  todos.forEach((item, index) => {
+    console.log(item);
+    const checkItem = item.completed ? 'checked' : '';
+    const lineTrought = item.completed ? 'trought-line' : '';
+    item.index = index;
+    console.log(item.description);
```

Comment:
Let's remove `console.log`

Reply:
Thank you for review my project. I improved these requested changes.